packages/utils/src/conversions.ts

```diff
@@ -220,3 +220,49 @@ export function bufferToHexadecimal(value: Buffer | Uint8Array): string {
 
     return Buffer.from(value).toString("hex")
 }
+
+/**
+ * Converts a buffer to a base64 string. It accepts 'Buffer' or 'Uint8Array'.
+ * @param value The buffer to convert.
+ * @returns The converted base64 string.
+ */
+export function bufferToBase64(value: Buffer | Uint8Array): string {
+    requireTypes(value, "value", ["Buffer", "Uint8Array"])
+
+    return Buffer.from(value).toString("base64")
+}
+
+/**
+ * Converts a base64 string to a buffer. This function does not check
+ * if the input value is a valid base64 string.
+ * @param value The base64 string to convert.
+ * @returns The converted buffer.
+ */
+export function base64ToBuffer(value: string): Buffer {
+    requireString(value, "value")
+
+    return Buffer.from(value, "base64")
+}
+
+/**
+ * Converts text (utf8) to a base64 string.
+ * @param value The text to convert.
+ * @returns The converted base64 string.
+ */
+export function textToBase64(value: string): string {
+    requireString(value, "value")
+
+    return Buffer.from(value, "utf8").toString("base64")
+}
+
+/**
+ * Converts a base64 string to text (utf8). This function does not check
+ * if the input value is a valid base64 string.
```

Comment:
As above, document what happens to invalid inputs.
In this case there are two different encodings which could go wrong.

Reply:
In this case `base64ToText` -> `textToBase64` gives a result that is different from the input. So I'll also mention the result may be unexpected.